src/database.js

```diff
@@ -99,6 +126,22 @@ export class Database {
     return factory ? factory.serializer : undefined; 
   }
 
+  randomRecord(collectionName) {
+    return this.randomRecords(collectionName)[0];
+  }
+
+  randomRecords(collectionName, limit = 1) {
+    const all = this.all(collectionName);
+    const records = [];
+
+    while (limit) {
```

Comment:
Mmmm... I don't think we'll have to do that because `--` suffix is executed **after** expression is evaluated, so loop is going to execute exactly `limit` times ;)